proto/dydxprotocol/accountplus/tx.proto

```diff
@@ -50,9 +51,10 @@ message MsgRemoveAuthenticatorResponse { bool success = 1; }
 // MsgSetActiveState sets the active state of the module.
 message MsgSetActiveState {
   option (amino.name) = "osmosis/smartaccount/set-active-state";
-  option (cosmos.msg.v1.signer) = "sender";
+  option (cosmos.msg.v1.signer) = "authority";
 
-  string sender = 1;
+  // Authority is the address that may send this message.
+  string authority = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
```

Comment:
just updated the name of this field to `Authority` for consistency with other gov controlled messages